spec/models/journaled/event_spec.rb

```diff
@@ -1,15 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe Journaled::Event do
+  let(:sample_journaled_event_class_name) { 'SomeClassName' }
   let(:sample_journaled_event_class) do
-    SomeClassName = Class.new do
+    Class.new do
       include Journaled::Event
     end
   end
 
-  after do
-    Object.send(:remove_const, :SomeClassName) if defined?(SomeClassName)
-    Object.send(:remove_const, :SomeModule) if defined?(SomeModule)
```

Comment:
I re-did some of the constant creation we were doing here with usages of `stub_const` so that this cleanup is taken care of for us!